rheem-platforms/rheem-flink/src/main/java/org/qcri/rheem/flink/compiler/criterion/RheemAggregator.java

```diff
@@ -0,0 +1,38 @@
+package org.qcri.rheem.flink.compiler.criterion;
```

Comment:
You might wanna consider renaming this class. First of all "Rheem" should not be part of any class name, and second, the name should be more descriptive of what type of aggregator represents. Also, are you sure this is the right package it should be placed in ?